meneco/meneco.py

```diff
@@ -47,116 +47,146 @@ def cmd_meneco(argv):
                         help='enumerate all minimal completions',
                         action='store_true',
                         default=False)
+    parser.add_argument('--json',
+                        help='produce JSON output',
+                        required=False, action='store_true', default=False)
 
     args = parser.parse_args(argv)
-    draft_sbml = args.draftnet
-    repair_sbml = args.repairnet
-    seeds_sbml = args.seeds
-    targets_sbml = args.targets
-
-    run_meneco(draft_sbml, seeds_sbml, targets_sbml,
-               repair_sbml, args.enumerate)
-
-def extract_xreactions(model, return_atom=True) :
-    lst = set(a[0] for pred in model if pred == 'xreaction' for a in model[pred])
-    if return_atom : 
-        atom = TermSet(Atom('xreaction(\"' +a[0]+'\",\"'+a[1]+'\")') for pred in model if pred == 'xreaction' for a in model[pred])
+
+    result = run_meneco(args.draftnet, args.seeds, args.targets,
+                        args.repairnet, args.enumerate, args.json)
+    if args.json:
+        print(json.dumps(result, indent=4))
```

Comment:
I added the indentation which is quite useful I think to ease the read when redirecting the output with the `--json` option